app/views/employees/index.html.slim

```diff
@@ -19,10 +19,10 @@
             td = employee.first_name
             td = employee.last_name
             td = employee.department
-            td 
+            td
               = link_to 'view', employee_path(employee.id), class: 'btn btn-primary'
               = link_to 'Edit', edit_employee_path(employee), class: 'btn btn-primary'
-              = link_to 'Delete', employee, data: { :confirm => 'Are you sure?' } , :method => :delete, class: 'btn btn-primary'
+              = link_to 'Delete', employee, data: {:confirm=>'Are you sure?'}, :method => :delete, class: 'btn btn-primary'
```

Comment:
It's nice to have those other spaces, it was just the space off the back that need to come off.
{ :confirm => 'Are you sure?' },